examples/stream.py

```diff
@@ -0,0 +1,18 @@
+"""Demonstrates musekafka.consumer.App in stream mode.
+
+To use, first start up the Kafka cluster using `make start-dev-kafka-cluster`.
+Next, start this example by running `./devenv python examples/stream.py`.
+Finally, use a tool such as kafkacat (https://github.com/edenhill/kafkacat) to send
+messages to the broker.
+"""
+
+import muselog
+
+from musekafka import consumers
+
+muselog.setup_logging(module_log_levels={"musekafka": "DEBUG"})
+
+app = consumers.App("stream_example", ["localhost:9092"], ["stream_example"])
+with app.stream() as messages:
```

Comment:
If I wanted to recreate the behavior of a constant streaming consumer I would just do `app.stream(close=False)` right?

Reply:
What do you mean by 'constant streaming'?

Reply:
An indefinite loop of streaming messages from a topic, (i.e: job search indexer, data-consumer, google-indexer...etc). The description of the close parameter made me think that it would by default stop streaming when it was fully caught up.

Reply:
`close` has nothing to do with stopping consumption. It only means to close the consumer after consumption is complete. Streaming is complete when `count` (for stream), `batch_count` (for batch), or `timeout` (for both) are hit. If you set count/batch_count and timeout to None, which is the default, you will get continuous streaming.

Reply:
Ah gotcha ok, thanks for explaining